Scripts/Solo/AutoRebuild.cfg

```diff
@@ -0,0 +1,9 @@
+# Config settings for AutoRebuild script.
+
+enabled = true # bool - Script will be removed if AutoRebuild has been disabled.
+
+check_x_tick = false # bool - Do we check to rebuild every x amount of ticks (can cause stutters)
+check_timer = 90 # s32 - how often is x for rebuild tick check
```

Comment:
I suggest removing check_x_tick and making it so check_timer = 0 disables it instead

Reply:
done